sdk/metricsadvisor/ai-metrics-advisor/src/models.ts

```diff
@@ -179,10 +192,11 @@ export type DataFeedRollupSettings =
  */
 export type DataFeedMissingDataPointFillSettings =
   | {
-      fillType: "SmartFilling" | "PreviousValue" | "NoFilling";
+      fillType: "SmartFilling" | "PreviousValue" | "NoFilling" | string;
+      customFillValue?: number;
     }
   | {
-      fillType: "CustomValue";
+      fillType: "CustomValue" | string;
```

Comment:
```suggestion
      fillType: "CustomValue";
```
Removing `| string` for custom case. It now looks similar to what is done to granularity type